update-bom/entrypoint.sh

```diff
@@ -44,23 +44,23 @@ fi
 
 if [ -f $CATALOG_FILE ]; then
     echo "Updating version catalog..."
-    if [ ! -z $bomProperty ]; 
+    if [ ! -z $bomProperty ];
     then
-        catalogProperty=$(echo $bomProperty | sed -r 's/([a-z0-9])([A-Z])/\1-\L\2/g' | sed -r 's/-version$//g')
+        catalogProperty=$(echo $bomProperty | sed -r -E 's/([a-z0-9])([A-Z])/\1-\L\2/g' | sed -r 's/-version$//g' | sed -r 's/elastic-search/elasticsearch/g')
```

Comment:
Can we do the rename in micronaut-elasticsearch or in the catalog or in any other place so we don't need to add here a specific case for it? I'm worried of needed more special cases for other modules...

Reply:
We can. I wouldn't rename the catalog entry, though, since it's more consistent than what we have currently (version property capitalized differently).

Reply:
Just to double check, this is the culprit, right? https://github.com/micronaut-projects/micronaut-elasticsearch/blob/d7d3a4e42865047ed883e457513154d744ef7f14/gradle.properties#L18 
Should it be `micronautElasticsearchVersion` (with lowecase `s` in Search)?

Reply:
Yes. For backwards compatibility we should probably leave it as is. As a workaround I added the ability to declare the name of the property in the catalog file.

Reply:
Why isn't enough to rename `micronautElasticSearchVersion` to `micronautElasticsearchVersion`? That name is for internal use only and with that change it will be converted to `micronaut-elasticsearch` for the catalog.

I'm missing something... :thinking:

Reply:
Right, maybe that can happen, not sure. But for this specific case in Elasticsearch I think we can rename the internal property and everything will work (famous last words :sweat_smile:)

Reply:
Thinking more about it it would probably work since this is already in place.

Reply:
Submitted PR with the change: https://github.com/micronaut-projects/micronaut-elasticsearch/pull/230

Reply:
Right. In any case I think this PR is good to go as is, since at some point we will want to migrate modules from using "guess based" names to actually properties which tell what is the name in the catalog, like this `catalogProperty`. So supporting both looks fine to me.

Reply:
Agree :+1: